README.md

```diff
@@ -29,12 +29,15 @@ Clone the Github repo and run the script directly from the local directory:
 ### sev-snp-measure
 ```
 $ sev-snp-measure --help
-usage: sev-snp-measure [-h] [--version] [-v] --mode {sev,seves,snp,snp:ovmf-hash,snp:svsm}
-                       [--vcpus N] [--vcpu-type CPUTYPE] [--vcpu-sig VALUE] [--vcpu-family FAMILY]
-                       [--vcpu-model MODEL] [--vcpu-stepping STEPPING] [--vmm-type VMMTYPE] --ovmf
-                       PATH [--kernel PATH] [--initrd PATH] [--append CMDLINE] [--guest-features VALUE]
-                       [--output-format {hex,base64}] [--snp-ovmf-hash HASH] [--dump-vmsa]
-                       [--vars-size VARS_SIZE] [--svsm SVSM]
+usage: sev-snp-measure [-h] [--version] [-v] --mode
+                       {sev,seves,snp,snp:ovmf-hash,snp:svsm} [--vcpus N]
+                       [--vcpu-type CPUTYPE] [--vcpu-sig VALUE]
+                       [--vcpu-family FAMILY] [--vcpu-model MODEL]
+                       [--vcpu-stepping STEPPING] [--vmm-type VMMTYPE] --ovmf
+                       PATH [--kernel PATH] [--initrd PATH] [--append CMDLINE]
+                       [--guest-features VALUE] [--output-format {hex,base64}]
+                       [--snp-ovmf-hash HASH] [--dump-vmsa] [--svsm PATH]
+                       [--vars-size SIZE | --vars-file PATH]
 
```

Comment:
nit: I use the following to add the usage message to the README:

```
COLUMNS=100 python3 ./sev-snp-measure.py --help
```

Hopefully this way you'll have more precise diffs.

(maybe I should document this somewhere or automate it in a Makefile task. hmm.)

Reply:
Ah, thanks for the hint!
I'll update it.

Reply:
`COLUMNS=100` :+1:

But I still get some unexpected changes.

Reply:
Looks OK to me.